### ISSUE TITLE
Create mock api and run as mock server from the endpoint response

### ISSUE THREAD

**Issue by rajani007** [a85/POSTMan-Chrome-Extension#12867]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

No. There is no problem as such. It's just good to have a feature.

### Describe the solution you'd like

Let's say we have a service that works by hitting any environment, but due to some network or VPN limitations that won't work when you are out of an enterprise environment. In that case, it would be good to run the same service as the mock server and continue with the development. 

In short, a mock server of a given API. 

### Describe alternatives you've considered

I use Mockoon for REST APIs or sometimes if it is a SOAP service, I use the SOAP UI in-built feature.
